tools/azure_dependencies.sh

```diff
@@ -9,7 +9,7 @@ if [ "${TEST_MODE}" == "pip" ]; then
 	python -m pip install $STD_ARGS --only-binary="numba,llvmlite,numpy,scipy,vtk,dipy" -e .[test,full] "numpy<2"
 elif [ "${TEST_MODE}" == "pip-pre" ]; then
 	${SCRIPT_DIR}/install_pre_requirements.sh
-	python -m pip install $STD_ARGS --pre -e .[test]
+	python -m pip install $STD_ARGS --pre -e .[test_extra]
```

Comment:
the corresponding branch in `github_actions_dependencies.sh` uses `test_extra` too, and it's needed for `utils/tests/test_config.py` (I think previously it was passing on windows by chance)